scripts/update_json_docs.sh

```diff
@@ -23,25 +29,33 @@ function pushDocs () {
     echo "Nothing to commit."
   fi
   cd ..
+
+  echo "Done deploying JSON documentation."
 }
 
 function cleanSubmodule () {
-    echo "Cleaning up..."
+    echo "Cleaning up!"
     git submodule deinit -f ghpages
     git reset HEAD .gitmodules
     git reset HEAD ghpages
     rm -rf ghpages
     rm -f .gitmodules
     rm -rf .git/modules/ghpages
+    echo "Done cleaning up!"
 }
 
+# Run this to verifiy that the docs build successfully.
+buildDocs $TRAVIS_BRANCH
```

Comment:
Can you use `${TRAVIS_BRANCH}` here and elsewhere? (There is a `bash` lint tool I used when making `bash` scripts for this project.)